tests/staggered_dslash_ctest.cpp

```diff
@@ -569,9 +212,9 @@ TEST_P(StaggeredDslashTest, benchmark) {
 #endif
     for (int p = 0; p < pmax; p++) {
```

Comment:
Can you move this initialization code into a custom routine in `StaggeredDslashTestWrapper` called something like `ctest_initialize_once`? The intention is just to move all the logic for initializing these backup fields into the test wrapper.

Reply:
It's important that it doesn't go into `init_ctest`---this should be called _exactly_ once, not once per test.